Modules/hpe3par_remote_copy.py

```diff
@@ -389,12 +389,78 @@ def recover_remote_copy_group(
     return (True, True, "Recovered Remote Copy Group %s successfully." % remote_copy_group_name, {})
 
+def admit_remote_copy_links(
+            client_obj,
+            storage_system_username,
+            storage_system_password,
+            storage_system_ip,
+            target_name,
+            source_port,
+            target_port_wwn_or_ip
+            ):
+    if storage_system_username is None or storage_system_password is None:
+        return (
+            False,
+            False,
+            "Admit remote copy link failed. Storage system username or password is null",
+            {})
+    if target_name is None:
+        return (False, False, "Admit remote copy link failed. Target name is null", {})
+    if source_port is None:
+        return (False, False, "Admit remote copy link failed. Source port address is null", {})
+    if target_port_wwn_or_ip is None:
+        return (False, False, "Admit remote copy link failed. Target port WWN/IP is null", {})
+    if storage_system_ip is None:
+        return (False, False, "SSH to 3par storage system failed. Storage system IP address is null", {})
+    try:
+        client_obj.login(storage_system_username, storage_system_password)
+        client_obj.setSSHOptions(storage_system_ip, storage_system_username, storage_system_password)
+        response = client_obj.admitRemoteCopyLinks(target_name, source_port, target_port_wwn_or_ip)
+    except Exception as e:
+        return (False, False, "Admit remote copy link failed | %s" % (e), {})
+    finally:
+        client_obj.logout()
+    return (True, True, "Admit remote copy link %s successful." % port_address, {})
+
+def dismiss_remote_copy_links(
+            client_obj,
+            storage_system_username,
+            storage_system_password,
+            storage_system_ip,
+            target_name,
+            source_port,
+            target_port_wwn_or_ip
+            ):
+    if storage_system_username is None or storage_system_password is None:
+        return (
+            False,
+            False,
+            "Dismiss remote copy link failed. Storage system username or password is null",
+            {})
+    if target_name is None:
+        return (False, False, "Dismiss remote copy link failed. Target name is null", {})
+    if source_port is None:
+        return (False, False, "Dismiss remote copy link failed. Source port address is null", {})
+    if target_port_wwn_or_ip is None:
+        return (False, False, "Dismiss remote copy link failed. Target port WWN/IP is null", {})
+    if storage_system_ip is None:
+        return (False, False, "Dismiss remote copy link failed. Storage system IP address is null", {})
+    try:
+        client_obj.login(storage_system_username, storage_system_password)
+        client_obj.setSSHOptions(storage_system_ip, storage_system_username, storage_system_password)
+        response = client_obj.dismissRemoteCopyLinks(target_name, source_port, target_port_wwn_or_ip)
```

Comment:
Similar to above, what happens if the link is not admitted at all and we try to remove it?